draft-ietf-core-echo-request-tag.md

```diff
@@ -138,15 +138,15 @@ The Echo option MAY be included in any request or response (see {{echo-app}} for
 
 The application decides under what conditions a CoAP request to a resource is required to be fresh. These conditions can for example include what resource is requested, the request method and other data in the request, and conditions in the environment such as the state of the server or the time of the day.
 
-If a certain request is required to be fresh, the request does not contain a fresh Echo option value, and the server cannot verify the freshness of the request in some other way, the server MUST NOT process the request further and SHOULD send a 4.01 Unauthorized response with an Echo option. The server MAY include the same Echo option value in several different responses and to different clients.
+If a certain request is required to be fresh, the request does not contain a fresh Echo option value, and the server cannot verify the freshness of the request in some other way, the server MUST NOT process the request further and SHOULD send a 4.01 Unauthorized response with an Echo option. The server MAY include the same Echo option value in several different response messages and to different clients. One example of this is in the case of event-based freshness (see later in this section) with no event taking place between the responses.
```

Comment:
I think it is more important to have an example of time based to not give the idea that only event based echo values can be reused. The server is perfectly fine to send the same echo value to to clients whos requests came close in time. This saves some processing.